pages/sight.vue

```diff
@@ -35,30 +50,62 @@
         </v-card>
       </v-container>
       <spacer gap="3" />
-      <div class="carosel">
-        <v-carousel hide-delimiters>
-          <v-carousel-item
-            v-for="(item, i) in items"
-            :key="i"
-            :src="item.src"
-          ></v-carousel-item>
-        </v-carousel>
-      </div>
+
+      <v-card height="600">
+        <v-tabs
+          v-model="tab"
+          height="100"
+          background-color="primary"
+          dark
+          centered
+        >
+          <v-tab
+            v-for="item in items1"
```

Comment:
Images are not looking good!

Reply:
These images were provided by the sight coordinators. I think we should keep them for some time.

Reply:
Modified the size of the images.